example_config.yaml

```diff
@@ -13,6 +13,9 @@ chihaya:
   udp:
     addr: 0.0.0.0:6881
     allow_ip_spoofing: false
+    max_clock_skew: 10s
+    private_key: |
+      paste a random string here that will be used to hmac connection IDs
```

Comment:
Why is this on two lines and why is the second line not a comment? YAML specific?